## Задание 1

**Если мы рассчитали 95% доверительный интервал для среднего значения, то какие из следующих утверждений являются верными?**

* Среднее значение в генеральной совокупности точно превышает нижнюю границу 95% доверительного интервала.
* Среднее значение в генеральной совокупности точно принадлежит рассчитанному доверительному интервалу.
* Если многократно повторять эксперимент, то 95 % выборочных средних значений будут принадлежать рассчитанному нами доверительному интервалу._(интервалы построены с выборочными по середине)_
* **Мы можем быть на 95% уверены, что среднее значение в генеральной совокупности принадлежит рассчитанному доверительному интервалу.**
* **Если многократно повторять эксперимент, для каждой выборки рассчитывать свой доверительный интервал, то в 95 % случаев истинное среднее будет находиться внутри доверительного интервала.**

## Задание 2

**Если бы мы увеличили объем выборки в два раза (при условии, что показатель стандартного отклонения остался прежним), то 95% доверительный интервал:**

* стал более широким
* **стал более узким**
* возможны оба варианта

## Задание 3

**В центре 95% доверительного интервала, рассчитанного по выборочным значениям, находится:**

* Среднее значение генеральной совокупности
<br>_(интервал пытается словить его, интервал - когда не известно mean ГС)_
* **Выборочное среднее значение**
* Значение стандартной ошибки среднего

## Задание 4

**Рассчитайте 99% доверительный интервал для следующего примера:**

In [18]:
x = 10
sd = 5
n = 100
z = 2.576 #это граница интервала а не z-score

In [19]:
import numpy as np

In [20]:
SE = sd / np.sqrt(100)

In [27]:
di_p = x + (z * SE)
di_p

11.288

In [29]:
di_m = x - (z * SE)
di_m

8.712

* [6,71 ; 13,29]
* [7,71 ; 12,29]
* [9,71 ; 12,29]
* **[8,71 ; 11,29]**


## Задание 5

**Укажите, какие утверждения относительно p-уровня значимости являются верными. Правильными могут быть все утверждения, некоторые, одно, а может не быть вообще!**

* Чем меньше p уровень значимости, тем сильнее получаемые различия. Например, если бы p уровень значимости в нашем исследовании был бы равен 0.0001, значит новый препарат еще сильнее бы влиял на скорость выздоровления
<br>_**(p-val не вл на степень эффекта)**_
* Вероятность случайно получить такие различия равняется 0.003
<br>_**(да, случайно получить=H0true, но! не такие, а такие или бОльшие)**_
* **ВCЕ ЭТИ УТВЕРЖДЕНИЯ НЕ ВЕРНЫ.**
* Т.к. в нашем исследовании p уровень значимости равен 0.003, то вероятность того, что верна нулевая гипотеза (новый препарат не влияет на скорость выздоровления _**(=эффект случаен)**_) также равняется всего-лишь 0.003
<br>_**(same as prev wrong)**_
* Если бы в исследовании мы получили p=0.9, это означало бы, что верна нулевая гипотеза, и новый препарат не влияет на скорость выздоровления
<br>_**(мы не можем принять H0, мы можем только не смочь отклонить)**_
* Статистически значимый результат, всегда означает ценный и осмысленный результат
<br>_**(вообще не означает)**_


## Задание 6

**При p<0.05 нулевая гипотеза обычно отклоняется, и различия считаются статистически достоверными. Однако часто в статистике используется более жесткий критерий достоверности различий, например, при условии, что p<0.01.**

**Значение p-уровня значимости, которое выбирается в качестве порога, обозначается буквой α (альфа). Например, если исследователь решил, что α=0.05, то и нулевая гипотеза будет отклоняться при условии, что p<0.05.**

**Если в определенной ситуации весьма рискованно отклонить нулевую гипотезу, когда она на самом деле верна, то лучше использовать показатель α равный:**

<br>_(Меньше p - больше оснований отклонить)_
<br>_(риск = отклонить должно быть сложно - порог должен быть ниже)_
<br>_но не понятно соразмерна ли степень риска с высотой порога_
<br>_"But the threshold depends on your field of study – some fields prefer thresholds of 0.01, or even 0.001."_
* 0.05
* **0.001**
* 0.1
* 0.5


## Задание 7

**Использование доверительных интервалов зачастую рассматривают как альтернативный способ проверки гипотез. В нашем случае, если значение 20 (предполагаемое среднее значение в генеральной совокупности) не будет принадлежать 95% доверительному интервалу, рассчитанному по выборочным данным, то у нас будет достаточно оснований отклонить нулевую гипотезу.**

**Проверьте согласуются ли результаты двух этих подходов: рассчитайте 95% доверительный интервал для среднего значения, на примере с тестированием нового препарата.  Параметры: n=64,sd=4, X=18.5**

In [30]:
n = 64
sd = 4
X = 18.5
z = 1.96

In [31]:
se = sd / np.sqrt(n)

In [35]:
di_p = X + (z * se)
di_p

19.48

In [34]:
di_m = X - (z * se)
di_m

17.52

In [ ]:
АЛЬТ РЕШ
x-M/se = -1.5/0.5 = -3
p(val) = 0.0038 < 0.005 

доверительным решать проще если 0.05 0.01

20 не принадлежит доверительному интервалу [17.52, 19.48], поэтому нулевая гипотеза H0 отклоняется.

* 20 принадлежит доверительному интервалу — отклоняем Н0
* **20 не принадлежит доверительному интервалу — отклоняем Н0**
* 20 не принадлежит доверительному интервалу — отклоняем Н1
* 20 принадлежит доверительному интервалу — отклоняем Н1

## Задание 8

**Данные некоторого исследования сообщают нам, что средний рост детей в 10 лет составляет 136 сантиметров. Однако это лишь выборочная оценка, и исследователи рассчитали 99% доверительный интервал, который составил [130, 142]. Укажите верные утверждения:**


* **У нас достаточно оснований отклонить нулевую гипотезу, что среднее в генеральной совокупности равняется 128**
<br>_**(128 не попало в интервал)**_
* Доверительный интервал не может иметь такие границы, т. к. выборочное стандартное отклонение равняется 10 _**(ст.откл=(142-130)/(2*2.6)=2.3)**_, следовательно доверительный интервал должен быть значительно шире
* Вероятность того, что истинное среднее значение больше 142, составляет 0.01 _**(0.005)**_
* **У нас достаточно оснований отклонить нулевую гипотезу, что среднее в генеральной совокупности равняется 143**


## Задание 9

**Предположим, нулевой гипотезой вашего исследования являлось предположение, что среднее в генеральной совокупности равняется 100. Вы получили p = 0.12 и не смогли отклонить нулевую гипотезу. Однако позже выяснилось, что среднее в генеральной совокупности равняется 114. Как можно описать результаты данного исследования?**

- I рода - _**(false pos = false эффект = false отклH0)**_ 
_**кстати откл H0 = приняли H1**_
- II рода - false neg

* **Вы совершили ошибку второго рода**
* Вы совершили ошибку первого рода
* Вы не совершали ни ошибку первого рода, ни ошибку второго рода


## Задание 10

**В среднем слушатели курса по введению в статистику набирают 115 баллов, однако, в 2015 году средний балл случайно выбранных 144 участников составил 118 со стандартным отклонением равным 9. Рассчитайте p-уровень значимости для проверки нулевой гипотезы о том, что среднее значение баллов в 2015 году равняется 115.**

* p < 0.05 – принимаем нулевую гипотезу
* p > 0.05 – принимаем нулевую гипотезу
* p > 0.05 – отклоняем нулевую гипотезу
* **p < 0.05 – отклоняем нулевую гипотезу**

In [9]:
import scipy.stats as ss
import numpy as np
(1-ss.norm.cdf((118-115)/(9/np.sqrt(144))))*2

6.334248366623996e-05